Wangscape/logging/Level.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+namespace logging
+{
+
+
+enum class Level
+{
+    Debug,
+    Info,
+    Warning,
+    Error
```

Comment:
I agree, "developer debugging messages" should be available to users so they can make bug reports. But they will be irrelevant noise when they are not encountering a bug in Wangscape, so they should be optional. That's why I think the log level should be configurable in the command line or options file.